.gitmodules

```diff
@@ -0,0 +1,24 @@
+[submodule "sc-flexy-datapoint-lib"]
+	path = sc-flexy-datapoint-lib
+	url = https://github.com/hms-networks/sc-flexy-datapoint-lib.git
+[submodule "flexy-azure-connector-generic"]
+	path = flexy-azure-connector-generic
+	url = https://github.com/hms-networks/flexy-azure-connector-generic.git
+[submodule "sc-flexy-historical-data-queue-lib"]
+	path = sc-flexy-historical-data-queue-lib
+	url = https://github.com/hms-networks/sc-flexy-historical-data-queue-lib.git
+[submodule "sc-flexy-string-lib"]
+	path = sc-flexy-string-lib
+	url = https://github.com/hms-networks/sc-flexy-string-lib.git
+[submodule "sc-flexy-tag-info-lib"]
+	path = sc-flexy-tag-info-lib
+	url = https://github.com/hms-networks/sc-flexy-tag-info-lib.git
+[submodule "sc-flexy-file-util-lib"]
+	path = sc-flexy-file-util-lib
+	url = https://github.com/hms-networks/sc-flexy-file-util-lib.git
+[submodule "sc-flexy-logger-lib"]
+	path = sc-flexy-logger-lib
+	url = https://github.com/hms-networks/sc-flexy-logger-lib.git
+[submodule "sc-flexy-json-lib"]
+	path = sc-flexy-json-lib
+	url = https://github.com/hms-networks/sc-flexy-json-lib.git
```

Comment:
Each submodule should probably refer to the branch master so it doesn't end up in detached HEAD state. See https://github.com/hms-networks/flexy-azure-connector/blob/master/.gitmodules